google/cloud/bigtable/mutation_batcher_test.cc

```diff
@@ -138,7 +139,7 @@ class MutationBatcherTest : public bigtable::testing::TableTestFixture {
   MutationBatcherTest()
       : TableTestFixture(
             CompletionQueue(std::make_shared<FakeCompletionQueueImpl>())),
-        batcher_(new MutationBatcher(table_)) {}
+        batcher_(absl::make_unique<MutationBatcher>(table_)) {}
```

Comment:
Did you intend to use `std::make_unique` since C++14 is the min now?

Reply:
I think we should, but I am punting on that discussion until later.  We decided not to go back and remove C++11-ism, at least not initially.